provider/kubernetes/kubernetes_test.go

```diff
@@ -310,6 +311,7 @@ func TestLoadIngresses(t *testing.T) {
 				},
 			},
 		},
+		TLSConfiguration: []*tls.Configuration{},
```

Comment:
Is it needed to specify an empty `TLSConfiguration` for tests unrelated to the TLS functionality?

If no, then I'd prefer to omit those here and further below.

Reply:
Because I used the empty slice it was necessary (nil slice != empty slice) but I didn't know that you could append to a nil slice without any errors.